src/util/check_file.c

```diff
@@ -90,14 +90,16 @@ static errno_t perform_checks(const char *filename,
     }
 
     if (uid != (uid_t)(-1) && stat_buf->st_uid != uid) {
-        DEBUG(SSSDBG_TRACE_LIBS, "File '%s' must be owned by uid [%d].\n",
-              filename, uid);
+        DEBUG(SSSDBG_TRACE_LIBS,
```

Comment:
Please change commit subject of `DEBUG: changed verbosity of: ` it does not work well.

Reply:
Done.